### DISCUSSION TITLE
Github Pages not updating even after a couple hours

### DISCUSSION THREAD

**Discussion by rahulharikumarr** [community/community#109667] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have a GitHub page repo in which I have links to other webpages I've hosted. Recently I updated it with links, however it has been at least 4 hours and the changes haven't reflected in the webpage yet. Is it longer than usual? I just want to know if I should be looking into it further or waiting.

**Comment by namith1003** [comment]
It does take really long sometimes, maybe by the time I'm writing this it might have updated, if not try to give a bit more detail and I will try and help.
